### ISSUE TITLE
Treat default filter as isFilterActive = false

### ISSUE THREAD

**Issue by juniwalk** [contributte/datagrid#1082]
I'd like to propose that filter is not considered active when it is in it's default state. Force filter active would still override that though.

```diff
/********************************************************************************
 *                                  FILTERING *
 ********************************************************************************/
public function isFilterActive(): bool
{
+	if ($this->filter === $this->defaultFilter && !$this->forceFilterActive) {
+		return false;
+	}

	$is_filter = ArraysHelper::testTruthy($this->filter);

	return $is_filter || $this->forceFilterActive;
}
```

**Comment by juniwalk** [comment]
@radimvaculik If this is stupid, could be add method `isFilterDefault()` (or something) to determine if grid is on its defaults?

**Comment by juniwalk** [comment]
@radimvaculik So should I prepare another PR? To Which branch?

**Comment by radimvaculik** [comment]
`next` branch 🙏

**Comment by juniwalk** [comment]
@radimvaculik Could you merge #1081 before I start working on this please?

**Comment by juniwalk** [comment]
Fixed in #1084
